pkg/heat/dbsync.go

```diff
@@ -40,7 +40,7 @@ func DBSyncJob(
 	envVars["KOLLA_CONFIG_STRATEGY"] = env.SetValue("COPY_ALWAYS")
 	envVars["KOLLA_BOOTSTRAP"] = env.SetValue("true")
 
-	volumes := GetVolumes(ServiceName)
+	volumes := GetVolumes(ServiceName, ServiceName)
```

Comment:
Should this be: `GetVolumes(ServiceName, instance.Name)` ?

I understand that in case of `dbsync` you only need parent (=heat) volumes, but in case (that probably won't never happen) the CR has a different name, you might fail passing the secret.
This [1] is an example to give you the idea of what I was just describing.

[1] https://github.com/openstack-k8s-operators/manila-operator/blob/main/pkg/manilaapi/statefulset.go#L71

Reply:
Yeah, seems reasonable. Updated now.